docs/release-playbook.md

```diff
@@ -0,0 +1,205 @@
+# Bazel Release Playbook
+
+Authors: lpino@, pcloudy@, laurentlb@
+
+Status: Work in progress
+
+Last Updated: 2018-11-09
+
+Link to this document: go/bazel-release-playbook
+
+Congratulations, you're the new release manager!
+
+Each release has a tracking bug (see the list on http://go/bazel-release). The bug
+includes a "Target RC date". On that day, create a new release candidate.
```

Comment:
Outdated. The list of releases is here: https://github.com/bazelbuild/bazel/issues?utf8=%E2%9C%93&q=label%3Arelease+

Reply:
Done.